examples/azure-vnet-to-vnet-peering/README.md

```diff
@@ -0,0 +1,20 @@
+# VNET to VNET Peering
+
+This template creates two VNETs in the same location, each containing a single subnet, and creates connections between them using VNET Peering.
+
+## main.tf
```

Comment:
Yes! This was a recent idea that we had, and we plan to do that with most of them in the Hashicorp PRs if time permits.